css/style.css

```diff
@@ -0,0 +1,100 @@
+
+
+body {
+	font-family: Arial;
+	background: black;
+  -webkit-font-smoothing: antialiased;
+}
+
+hgroup { 
+	text-align:center;
+	margin-top:0;
+    color: black;
+}
+
+h1, h3 { font-weight: 300; }
+
+h1 { color: gainsboro; }
+
+h3 { color: #4a89dc; }
+
+form {
+	width: 380px;
+	margin: 7em auto;
+	padding: 3em 2em 2em 2em;
```

Comment:
This particular rule can be written in a slightly more concise manner. Do you know how?